initializers/racks.yml

```diff
@@ -16,6 +16,7 @@
 #     text_field: Description
 # - site: SING 1
 #   name: rack-03
+#   group: SING 1
```

Comment:
I like this addition, though I believe the value should be `cage 101`, shouldn't it?

Reply:
You are absolutely right, my bad. Fixed.